### PR TITLE
Feature/eui 132

### PR DESCRIPTION
**Pull request by adnanakgun**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-132

### Change description ###

Made requested changes on the ticket

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: api/auth/index.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export async function attach(req: EnhancedRequest, res: express.Response, next:
             req.auth.token = jwt
             req.auth.userId = userId
             req.auth.expires = expires
-            req.auth.roles = roles
+            req.auth.roles = [roles]
```
Comment (smogese) [inline api/auth/index.ts]: Why?

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "start:node": "cd api && npm run watch",
     "test:node": "cd api && npm run test",
     "test:node:watch": "cd api && npm run test:watch",
-    "test:coverage:ng": "ng test --code-coverage --watch",
+    "test:coverage:ng": "ng test rpa-pui-manager --code-coverage --watch",
```
Comment (smogese) [inline package.json]: Don't do covergage like this, put the config in angular.json so its part of ng test.
The reason for this is that the pipeline runs the test command only and expects coverage reports to be generated like that. See JUI for example

File: src/auth/mock/user.mock.ts [hunk 3]
```diff
@@ -20,4 +20,4 @@ export const userMock: UserInterface = {
     userId: 'Something Seomthing',
   }
```
Comment (smogese) [inline src/auth/mock/user.mock.ts]: What are you mocking and why?

Comment (adnanakgun) [reply src/auth/mock/user.mock.ts]: Mocking the endpoint as there is none there yet

File: api/auth/index.ts (hunk 1 above)
Comment (adnanakgun) [reply api/auth/index.ts]: Because it expects an array.

File: package.json (hunk 2 above)
Comment (adnanakgun) [reply package.json]: I've added `"codeCoverage": true,` to the angular.json file now, but I won't be removing it from here as the task name does indicates code coverage.

File: api/auth/index.ts (hunk 1 above)
Comment (adnanakgun) [reply api/auth/index.ts]: Changed it back to none array.

File: angular.json [hunk 4]
```diff
@@ -78,6 +78,11 @@
             "assets": [
               "src/favicon.ico",
               "src/assets"
+            ],
+            "codeCoverage": true,
```
Comment (adnanakgun) [inline angular.json]: See here @smogese

**Pull request merged**